packages/odata-service-inquirer/src/utils/store.ts

```diff
@@ -11,3 +13,22 @@ export async function getBackendSystemService(): Promise<Service<BackendSystem,
     });
     return backendService;
 }
+
+/**
+ * Fetch all backend systems.
+ *
+ * @param includeSensitiveData - whether to include sensitive data
+ * @returns backened systems
+ */
+export async function getAllBackendSystems(includeSensitiveData = false): Promise<BackendSystem[] | []> {
```

Comment:
Does this mean, if theres an issue with one system then no systems will be returned from the store? I think this should be fixed in the store module perhaps?

Reply:
If the JSON is malformed and can't be parsed, or in the case of why this issue was raised, I suspect the user does not have permissions to access the `systems.json` file, then I don't think we can have any improvement in terms of granularity.